kubeadm/scripts/PrepareNode.ps1

```diff
@@ -138,11 +99,7 @@ $newPath = "$global:NssmInstallDirectory;" +
 Write-Host "Registering kubelet service"
 nssm install kubelet $global:Powershell $global:PowershellArgs $global:StartKubeletScript
 
-if ($ContainerRuntime -eq "Docker") {
-    nssm set kubelet DependOnService docker
-} elseif ($ContainerRuntime -eq "containerD") {
-    nssm set kubelet DependOnService containerd
-}
+nssm set kubelet DependOnService containerd
```

Comment:
The kubelet service script needs to be aware of which networking it is using due to Calico issues, see https://github.com/projectcalico/calico/pull/6656/files